opentuner/resultsdb/models.py

```diff
@@ -59,6 +86,7 @@ class ProgramVersion(Base):
   program_id = Column(ForeignKey(Program.id))
   program = relationship(Program, backref='versions')
   version = Column(String(128))
+  parameter_info = Column(Text)
```

Comment:
Should this go on the TuningRun?  I could imagine you tuning the same version of the program with different search space representations.